src/shogun/mathematics/linalg/LinalgBackendEigen.h

```diff
@@ -0,0 +1,48 @@
+#include <shogun/mathematics/eigen3.h>
```

Comment:
mmm this header (`LinalgBackendEigen.h`) will never be included anywhere outside of `linalg` implementation right?